src/uwtools/scheduler.py

```diff
@@ -0,0 +1,117 @@
+"""
+Job Scheduling
+author: Ryan Long <ryan.long@noaa.gov>
+"""
```

Comment:
`git log` will save the name of the developer.
Also, it becomes unwieldy when many developers add their names to the list as time progresses.
The doc block (if present) should follow numpy protocol.